Modules/Filtering/Smoothing/include/itkBinomialBlurImageFilter.hxx

```diff
@@ -146,15 +146,15 @@ BinomialBlurImageFilter<TInputImage, TOutputImage>::GenerateData()
   typename TTempImage::IndexType indexShift;
 
   // How many times has the algorithm executed? (for debug)
-  int num_reps = 0;
+  itkDebugStatement(int num_reps = 0);
```

Comment:
Thanks Hans, but why not just removing this `num_reps` variable? Is the BinomialBlurImageFilter still actively being debugged, after so many years? (Looks like the filter was added to ITK more than 20 years ago: commit d236c7aa0ee7c361b2920c09ef9d170c583d5de2, Damion Shelton, June 7, 2001)